### ISSUE TITLE
[Feature request] ClearDepthStencil overload that lets you choose which to clear

### ISSUE THREAD

**Issue by Doom2fan** [mellinoe/veldrid#261]
I've noticed there's no way to clear only the depth buffer or only the stencil buffer, only both at once. This is a fairly useful feature to have, so it'd be nice to have it in Veldrid too.
(I actually need it for my engine, myself, so I can clear the stencil buffer but not the depth)

**Comment by mellinoe** [comment]
Agreed, this would be nice to have. It shouldn't be that hard to support. Given the existing semantics and documentation for `CommandList.ClearDepthStencil`, it probably makes sense to just add `ClearDepth` and `ClearStencil` methods instead of introducing a clear mask enum. Under the hood, though, each backend should probably just implement one function using their respective mask enums.
